python/repositories.bzl

```diff
@@ -241,6 +242,7 @@ exports_files(["python", "{python_path}"])
 py_runtime(
     name = "py3_runtime",
     files = [":files"],
+    coverage_tool = {coverage_tool},
```

Comment:
Our team's internal rules do something like
```starlark
config_setting(
    name = "coverage_enabled",
    values = {"collect_code_coverage": "true"},
)

py_runtime(
    name = "py3_runtime",
    coverage_tool = select({
        ":coverage_enabled": "{coverage_tool}",
        "//conditions:default": None,
    }),
    ...
)
```
so that bazel doesn't need to fetch the coverage repo in order to configure the toolchain except when running for code coverage, which is nice.

Reply:
Fixed this, thanks for the comment!